src/startupscriptgenerator/src/common/consts/ext_var_names.go

```diff
@@ -10,3 +10,4 @@ const PhpOriginEnvVarName string = "PHP_ORIGIN"
 const UserAppInsightsKeyEnv string = "APPINSIGHTS_INSTRUMENTATIONKEY"
 const UserAppInsightsEnableEnv string = "ApplicationInsightsAgent_EXTENSION_VERSION"
 const UserAppInsightsConnectionStringEnv string = "APPLICATIONINSIGHTS_CONNECTION_STRING"
+const PythonEnableGunicornMultiWorkersEnvVarName string = "PYTHON_ENABLE_GUNICORN_MULTIWORKERS"
```

Comment:
is this python_ something appservice team recommended? sounds very weird, but if they are recommending it .. it is what it is

Reply:
They recommend us to add a python prefix on it for future use, and we'll double check with them on next week's meeting.